### DISCUSSION TITLE
a few errors when running

### DISCUSSION THREAD

**Discussion by johnmolino** [timelyart/Kairos#118] [Q&A] [answered]
Hello perhaps someone can help....

I keep getting an error when it tries to either read my watchlist (c:\Kairos-3.17>python main.py screener_to_watchlist.yaml or c:\Kairos-3.17>python main.py browse.yaml )

Traceback (most recent call last):
  File "c:\Kairos-3.17\tv\tv.py", line 1198, in open_chart
    el_options = find_elements(browser, css_selectors['options_watchlist'])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Kairos-3.17\tv\tv.py", line 508, in find_elements
    elements = WebDriverWait(browser, delay).until(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\jm\AppData\Local\Programs\Python\Python312\Lib\site-packages\selenium-4.24.0-py3.12.egg\selenium\webdriver\support\wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message:

or this one for the screener_to_watchlist.yaml (seems like same error) perhaps TV changed something?

2024-09-10 20:04:24.798 ERROR tv - run: Message:
Stacktrace:
        GetHandleVerifier [0x00E58213+26163]
        (No symbol) [0x00DE9CC4]
        (No symbol) [0x00CE24C3]
        (No symbol) [0x00D27453]
        (No symbol) [0x00D2762B]
        (No symbol) [0x00D66B62]
        (No symbol) [0x00D4AD04]
        (No symbol) [0x00D64661]
        (No symbol) [0x00D4AA56]
        (No symbol) [0x00D1BE89]
        (No symbol) [0x00D1C8CD]
        GetHandleVerifier [0x0112D313+2996019]
        GetHandleVerifier [0x01181B89+3342249]
        GetHandleVerifier [0x00EE7AEF+614159]
        GetHandleVerifier [0x00EEF17C+644508]
        (No symbol) [0x00DF27FD]
        (No symbol) [0x00DEF6F8]
        (No symbol) [0x00DEF895]
        (No symbol) [0x00DE1C16]
        BaseThreadInitThunk [0x7731FCC9+25]
        RtlGetAppContainerNamedObjectPath [0x77C880CE+286]
        RtlGetAppContainerNamedObjectPath [0x77C8809E+238]

Traceback (most recent call last):
  File "c:\Kairos-3.17\tv\tv.py", line 3439, in run
    markets = get_screener_markets(browser, screener_yaml)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Kairos-3.17\tv\tv.py", line 3536, in get_screener_markets
    el_select = find_element(browser, css_selectors['select_screener'])
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Kairos-3.17\tv\tv.py", line 481, in find_element
    element = WebDriverWait(browser, delay).until(
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\jm\AppData\Local\Programs\Python\Python312\Lib\site-packages\selenium-4.24.0-py3.12.egg\selenium\webdriver\support\wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message:

Thank you,
John

**Comment by timelyart** [comment, marked as answer]
I will look into the errors. This may be a recent change in the website. 
I have referenced this question into a new issue #119. 

Next time when you encounter errors please open an issue instead of a discussion, thank you.

**Reply by johnmolino** [reply]
Thank you - please let me know if you need any other info.  yes I noticed that about the issues after I did this - it's been a while since I used github.  Apologies.  Feel free to delete this if you'd like.
